assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -93,13 +90,7 @@ describe( 'core/modules modules', () => {
 					{ body: {}, status: 200 }
 				);
 
-				registry.dispatch( STORE_NAME ).activateModule( slug );
-
-				// Wait until this activation action has completed.
-				await subscribeUntil( registry, () => registry
-					.select( STORE_NAME )
-					.isDoingSetModuleActivation( slug ) === false
-				);
+				await registry.dispatch( STORE_NAME ).activateModule( slug );
```

Comment:
Shouldn't we still wait here until module activation has been completed? There's the API call to refetch authentication data which is async.

Reply:
We do wait – the action itself is async by association with those internal calls and will return a promise here that resolves when the action including its internal async calls are completed, similar to a resolver.

Reply:
Oops - I had totally missed the `await` 🤦‍♂️